docs/coding-guidelines/project-guidelines.md

```diff
@@ -17,13 +17,13 @@ once before you can iterate and work on a given library project.
  - Build src\sign.proj
 
 ## Behind the scenes with build-test.cmd/sh
```

Comment:
Those scripts don't exist anymore, please delete the whole block. Same for build-packages.cmd/sh below.